### DISCUSSION TITLE
Docker Image Layout Directory

### DISCUSSION THREAD

**Discussion by camchambers** [foxglove/community#830] [Q&A] [answered]
[This feature](https://github.com/foxglove/studio/pull/6400) added the ability to override the default layout in the Docker image  and it seems to work well (thank you!).  

Are there plans to support preloading multiple layouts, or a directory of layouts? We're currently using [a fork](https://github.com/foxglove/studio/pull/5987) to do this, but it doesn't seem to be compatible with recent versions of Foxglove Studio.

**Comment by amacneil** [comment, marked as answer]
It's not currently planned, since multi-layouts in general we are continuing to evolve in our closed source product.

I would be curious to learn more about why you choose to use a docker image vs our hosted web service or official desktop app (since we already have that functionality)?

**Comment by camchambers** [comment]
This makes sense.  Thank you Adrian. I'll be sure to look into the data platform as an option.
